doc/src/neighbor.rst

```diff
@@ -55,14 +55,24 @@ For the *bin* style, the bin size is set to 1/2 of
 the largest cutoff distance between any pair of atom types and a
 single set of bins is defined to search over for all atom types.  This
 can be inefficient if one pair of types has a very long cutoff, but
-other type pairs have a much shorter cutoff.  For style *multi* the
-bin size is set to 1/2 of the shortest cutoff distance and multiple
-sets of bins are defined to search over for different atom types.
+other type pairs have a much shorter cutoff. The *multi* style uses
+different sized bins for collections of different sized particles. Different
```

Comment:
... sized particles, where "size" may mean the physical size of the particle or its cutoff distance for interacting with other particles.